ethiopic.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable @typescript-eslint/no-require-imports */
+/* eslint-disable no-undef */
+const ethiopic = require("./dist/cjs/ethiopic/index.js");
```

Comment:
Related to the todo "Understand the correct name, Ethiopian or Ethiopic?".
From what I searched online:
> ... “Ethiopic” can be used more broadly to describe anything related to Ethiopia, though it is less common in this usage compared to “Ethiopian".

IMO even if "Ethiopic" is technically more correct, using "Ethiopian" seems better because it results in better discoverability, due to it being more used.
The discoverability is also better even if someone searches for "Ethiopia", because the search would still match with the "Ethiopian" word.

Reply:
Edit: @temesgen-mulugeta agrees with Ethiopic:

> Thinking about the name, I think "Ethiopic" is better. It pays homage to the Geez language, which is the basis of the calendar.

Reply:
Thanks for the insights!
That is a good point to follow existing conventions, like Mac OS, and also ask other people who have more context.